app/controllers/application_controller.rb

```diff
@@ -83,6 +83,14 @@ def sms_acct_creation_notice(resource_name, sms_status)
     end
   end
 
+  def current_preference_set
```

Comment:
should call this `preference_set` and most likely just memoize.

Reply:
😕  think I created this helper early on but I don't have use fo it anymore. I'll just deleted it!